### PR TITLE
Use omp and driverify

### PR DESCRIPTION
**Pull request by Drup**
@aantron, @let-def : I wouldn't mind a review there. :)

### CODE REVIEW

**Comment by aantron** [comment]
Looks probably reasonable, but I have no experience with using OMP and Ocamlbuild/OASIS.

File: ppx/.merlin [hunk 1]
```diff
@@ -0,0 +1,2 @@
+REC
+FLG -open Migrate_parsetree -open Ast_403 -open Ppx_tools_403
```
Comment (mseri) [inline ppx/.merlin]: Do you mean 405 here?

**Comment by Drup** [comment]
@let-def You mean having a 406 version ? In the long term, sure. Right now, as long as it can run on 406 (which it should, thanks to omp), it's not a problem.

File: ppx/.merlin (hunk 1 above)
Comment (Drup) [reply ppx/.merlin]: Yes, thanks!

**Pull request merged**